File: CS/DiagnosticTool/Data/ItemBase.cs
using System.Xml.Serialization;

namespace DashboardDiagnosticTool.Data {
    public class ItemBase {
        [XmlAttribute]
        public int SessionId { get; set; }
        [XmlAttribute]
        public int ThreadId { get; set; }
        public ItemBase() { }
        public ItemBase(int sessionId, int threadId) {
            SessionId = sessionId;
            ThreadId = threadId;
        }
    }
}

File: VB/DiagnosticTool/Data/ItemBase.vb
Imports System.Xml.Serialization

Namespace DashboardDiagnosticTool.Data

    Public Class ItemBase

        <XmlAttribute>
        Public Property SessionId As Integer

        <XmlAttribute>
        Public Property ThreadId As Integer

        Public Sub New()
        End Sub

        Public Sub New(ByVal sessionId As Integer, ByVal threadId As Integer)
            Me.SessionId = sessionId
            Me.ThreadId = threadId
        End Sub
    End Class
End Namespace
